nav2_amcl/src/include/portable_utils.h

```diff
@@ -15,7 +15,7 @@ static double drand48(void)
   return ((double)rand()) / RAND_MAX;
 }
 
-static void srand48(long int seedval)
+static void srand48(int64 seedval)
```

Comment:
You should not change this

Reply:
I know it's a bit tricky. It complained:

 ``Use int16/int64/etc, rather than the C type long``

Reply:
This is here for Windows support, so I imagine that was done pretty intentionally to be a long int. You could add `// NOLINT` on the line to suppress it

Reply:
Shall I add the copyrights for Samsung research for this particular file?